setup.py

```diff
@@ -0,0 +1,4 @@
+import os
+os.system("julia install_packages.jl")
+if not os.path.exists('dataset'):
```

Comment:
This is a pure Julia project. Please don't combine it with Python.

Creating not existing path can be done in Jula too.

Reply:
Done. Should I create a new pull request? I'm sorry I'm new to GitHub. Also, the changes to the locations_pruned.yml were not made by me, but the code itself.

Reply:
What line of code made these changes?

Reply:
@logan-mo What line of code made these changes?

Reply:
@markub3327 No idea. I don't know Julia at all. But I do know If I run the code, the values in the locations_pruned are modified.

Reply:
![image](https://user-images.githubusercontent.com/63550599/224067940-a3db57b1-0421-48dd-998d-64be5615d4a8.png)
If I had to guess, this part that writes to the file probably does something.

Reply:
ok.